.github/workflows/pypi-package.yml

```diff
@@ -0,0 +1,26 @@
+name: Building & Publishing
+
+on:
+  push:
+    tags: 
+    - "v*"
+    - '!*dev*'
+    - '!*pre*'
+    - '!*post*'
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+
+  # Release to PyPI after a RELEASE gets PUBLISHED in @spacetelescope/astrocut/main
+  publish-to-pypi:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    name: Publish to PyPI
+
+    # We only want to upload to PyPI after a new release has been published on the @spacetelescope/astrocut repo
+    if: github.repository_owner == 'spacetelescope' && github.event_name == 'push'
+    with:
+      test_extras: test
+      test_command: pytest astrocut/tests/test*.py
```

Comment:
Don't we need:

```yml
    secrets:
      pypi_token: ${{ secrets.PYPI_PASSWORD_STSCI_MAINTAINER }}
```

in here? (or whatever the secret is called)

Reply:
It looks like there are 2 tokens - I'm guessing we just need the password, based off two examples here:

1. https://github.com/spacetelescope/gwcs/blob/e82d3fb937215c3b18582652740918a88ed9caa7/.github/workflows/build.yml#L15

2. https://github.com/spacetelescope/romancal/blob/a939846a68ed4362f50d6ae4c9317f7877a362f1/.github/workflows/build.yml#L23

I'll update accordingly.